### PR TITLE
chore(test): refactor atenlib test to make parameterization clearer

### PR DESCRIPTION
**Pull request by justinchuby**
Removed the class level parameterization

### CODE REVIEW

File: onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py [hunk 1]
```diff
@@ -1066,167 +1065,184 @@ def executor(function, args, kwargs):
...
-    add_decorate_info(
+class TestOutputConsistency_FullGraph(unittest.TestCase):
+    """Test output consistency between exported ONNX op run as a graph and PyTorch eager mode.
+
+    This is a parameterized test suite.
+    """
+
+    def setUp(self) -> None:
+        torch.manual_seed(42)
+        np.random.seed(42)
```
Comment (BowenBao) [inline onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py:1219]: ort random number

File: onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py [hunk 2]
```diff
@@ -965,9 +965,8 @@ def test_script_function_passes_checker(self, _, func_with_wrangler):
         onnx.checker.check_function(function_proto)  # type: ignore[attr-defined]
 
 
-def _graph_executor(test_class, outputs: Sequence[Any]):
+def _graph_executor(outputs: Sequence[Any]):
```
Comment (BowenBao) [inline onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py:968]: nit: return type annotation?

Comment (justinchuby) [reply onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py]: Done

File: onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py (hunk 1 above)
Comment (justinchuby) [reply onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py:1224]: Done

File: onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py [hunk 3]
```diff
@@ -1066,167 +1065,186 @@ def executor(function, args, kwargs):
...
-    ],
-    class_name_func=_get_test_class_name,
-)
-class TestOutputConsistency(unittest.TestCase):
-    """Test output consistency between exported ONNX models and PyTorch eager mode.
+def run_test_output_match(
+    test_suite: unittest.TestCase,
+    device: str,
+    dtype: torch.dtype,
+    op,
```
Comment (BowenBao) [inline onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py:1076]: nit type annotation

Comment (BowenBao) [reply onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py:1076]: if it's not an `OpInfo`, perhaps consider creating a Protocal

File: onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py [hunk 4]
```diff
@@ -1066,167 +1065,186 @@ def executor(function, args, kwargs):
...
-    common_device_type.instantiate_device_type_tests(
-        globals()[_test_class_name], globals(), only_for="cpu"
+    @common_device_type.ops(  # type: ignore[misc]
+        [info for info in OPS_DB if info.name in TESTED_OPS],
+        allowed_dtypes=TESTED_DTYPES,
     )
+    @unittest.skipIf(
+        version_utils.torch_older_than("2.0"), reason="only torch>=2.0 is supported"
+    )
+    def test_output_match_opinfo_(self, device: str, dtype: torch.dtype, op):
```
Comment (BowenBao) [inline onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py:1240]: nit: op here too for type annotations

**Review by BowenBao — APPROVED** [review]
🚢

File: onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py (hunk 4 above)
Comment (justinchuby) [reply onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py:1240]: Done

File: onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py (hunk 3 above)
Comment (justinchuby) [reply onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py]: Done

**Pull request merged**